### PR TITLE
feat(cli): add generate:comment command

### PR DESCRIPTION
**Pull request by Haroenv**
the command is exactly the same as generate:email, but it's in markdown, which makes the comments look a lot nicer and easier to copy-paste.

This can be used on PRs, issues etc.

### CODE REVIEW

File: deployer/src/index.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 print("\033[0m")
 print("")
 print("=======================")
-print("=  Deploy connectors  =")
+print("=  Deploy hjfgj  =")
```
Comment (Haroenv) [inline deployer/src/index.py]: 🤔

File: deployer/src/index.py [hunk 2]
```diff
@@ -39,21 +39,21 @@
...
-    print("Will be added :")
+    print("Will be \033[1;32madded\033[0m :")
     for config in added:
         added_log += " - " + config + "\n"
     print(added_log)
 
 if len(removed) > 0:
     print("")
-    print("Will be delete :")
+    print("Will be \033[1;31mdelete\033[0m :")
```
Comment (Haroenv) [inline deployer/src/index.py]: should be `deleted`

**Comment by s-pace** [comment]
Hi there,

Congratulations, your search is now ready!
I've successfully configured the underlying crawler and it will now run every 24h.

You're now a few steps away from having it working on your website:
- Copy the following CSS/JS snippets and add them to your page


<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />

<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: '36221914cce388c46d0420343e0bb32e',
  indexName: 'react',
  inputSelector: '### REPLACE ME ####',
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>

- Add a search input if you don't have any yet, and update the inputSelector value in the code snippet to a CSS selector that targets your input field.

- Optionally customize the look and feel by following the DocSearch documentation (https://community.algolia.com/docsearch/documentation/)
- You can also check your configuration in the github repo (https://github.com/algolia/docsearch-configs/blob/master/configs/react.json). Feel free to open pull requests!

Feel free to get back to us if you have any issues or questions regarding the integration.
We'd also be happy to get your feedback and thoughts about DocSearch - so we can continue to improve it.

Have a nice day :)

Output I have. Not working properly

**Comment by Haroenv** [comment]
Thanks @s-pace, will try it more. Doesn't make much sense it still has the original though ...

**Comment by maxiloc** [comment]
@Haroenv Do you want to continue working on this one, or we should close it ?

**Comment by Haroenv** [comment]
I don't use the tool so I don't have any skin in the game. Feel free to close if you think it's not worth it :)

**Pull request closed without merging**